packages/code-du-travail-frontend/pages/outils/[slug].js

```diff
@@ -69,10 +49,24 @@ class Outils extends React.Component {
 
 export default Outils;
 
-const { fonts } = theme;
+Outils.getInitialProps = async ({ query }) => {
+  // we don't request data from api since outils are client side only
```

Comment:
I think we can now remove this comment :)

Reply:
yup

Reply:
Iups this is a mistake of mine ! there should be no fetch in there

Reply:
we could use fetch to retrieve metadata (or use @cdt/data/outils.json)

Reply:
I'm currently moving outils.json to its own subrepo in @cdt/data so we can import it. Then I'll import it here so we can have tools working even when our api is down

Reply:
we finally do the multi source "outils" refactoring ^_^